torch/utils/_config_module.py

```diff
@@ -184,30 +194,34 @@ def _get_dict(
         config: Dict[str, Any] = {}
         for key in self._config:
             if ignored_keys and key in ignored_keys:
-                if skip_default and self._config[key] != self._default[key]:
+                if skip_default and not self._is_default(key):
```

Comment:
Yeah, so one of these use cases is for repro generation. I 100% don't  mind if I put a config in if a user wrote it, even if it's the "default".

Reply:
with updates to _is_default above, this should be fixed.